Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -2126,13 +2132,9 @@ private async static Task RemoveObjects_Test2(MinioClient minio)
                 {
                     tasks[i] = PutObject_Task(minio, bucketName, objectName + i.ToString(), null, null, 0, null, rsg.GenerateStreamFromSeed(5));
                     objectsList.Add(objectName + i.ToString());
-                    // Add sleep to avoid flooding server with concurrent requests
```

Comment:
Is this not required anymore?

Reply:
no, because only 50 objects are being uploaded